CHANGES.md

```diff
@@ -0,0 +1,17 @@
+# Changes in Version 3
+
+## Summary of breaking changes
+
+- `postgresql.conf` is templated instead of making in-line modifications to the distribution configuration
+- `CONNECT` and `public` schema `USAGE` privileges are explicitly granted to database users
+- `postgresql_users_databases` has been replaced by two variables which are both required.
```

Comment:
This ends with a period but the others don't.

Reply:
Removed `.`